Remove the native build and PicoCLI to allow support for multiple platforms (#56)
strimzi/drain-cleaner: README.md

## Deployment

> By default, the Drain cleaner will drain both Kafka and ZooKeeper pods. 

Reviewer: ```suggestion
By default, the Drain cleaner drains Kafka and ZooKeeper pods. 
```

> If you want to use this only to one of them, you can edit the Deployment set the `STRIMZI_DRAIN_KAFKA` or `STRIMZI_DRAIN_ZOOKEEPER` environment variables to `false`.

Reviewer: ```suggestion
If you want to use the Drain Cleaner with only one of them, you can edit the `Deployment` by setting the `STRIMZI_DRAIN_KAFKA` or `STRIMZI_DRAIN_ZOOKEEPER` environment variables to `false`.
```
